src/VirtualClient/VirtualClient.Core/Logging/SummaryFile/SummaryFileLoggerProvider.cs

```diff
@@ -27,17 +27,26 @@ public SummaryFileLoggerProvider(string filePath)
         }
 
         /// <summary>
-        /// Creates an <see cref="ILogger"/> instance that can be used to log events/messages
-        /// to an Application Insights endpoint.
+        /// Creates an <see cref="ILogger"/> instance that can be used write events to a 
+        /// summary log.
         /// </summary>
         /// <param name="categoryName">The logger events category.</param>
-        /// <returns>
-        /// An <see cref="ILogger"/> instance that can log events/messages to an Application
-        /// Insights endpoint.
-        /// </returns>
         public ILogger CreateLogger(string categoryName)
         {
-            SummaryFileLogger logger = new SummaryFileLogger(this.filePath);
+            string effectiveFilePath = this.filePath;
+
+            if (string.IsNullOrWhiteSpace(effectiveFilePath))
+            {
+                PlatformSpecifics platformSpecifics = VirtualClientRuntime.PlatformSpecifics
+                   ?? new PlatformSpecifics(Environment.OSVersion.Platform, RuntimeInformation.ProcessArchitecture);
+
+                string experimentId = VirtualClientRuntime.ExperimentId;
+                string logsPath = platformSpecifics.GetLogsPath();
+
+                effectiveFilePath = platformSpecifics.Combine(logsPath, $"{experimentId ?? DateTime.UtcNow.ToString("yyyy-MM-ddTHH-mm-ss-fff")}-summary.txt");
```

Comment:
We will always have an experimentId

Reply:
We will when running as VC proper but NOT when we are running as a unit test. The logic is really just a minor workaround. If you think it unnecessary, we can just use "summary.txt" if an experiment ID is not defined.

Reply:
I changed the logic to use the experiment ID if defined (typical VC runtime scenario).